x-pack/legacy/plugins/siem/public/lib/settings/__mocks__/use_timezone_setting.ts

```diff
@@ -4,8 +4,6 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { compose } from '../lib/compose/testing_compose';
-
-import { startApp } from './start_app';
-
-startApp(compose());
+export const useTimezoneBrowser = () => {
```

Comment:
One of those _fun_ `git rename`'s on this file from `apps/testing_app.ts` to `__mocks__/use_timezone_setting.ts`.